static/js/containers/pages/CheckoutPage_test.js

```diff
@@ -512,19 +524,67 @@ describe("CheckoutPage", () => {
       course_3: "run_3"
     })
   })
+  //
+  ;[true, false].forEach(isCourseRun => {
+    it(`selecting a course run ${shouldIf(
+      isCourseRun
+    )} update the product`, async () => {
+      basket = makeBasketResponse(
+        isCourseRun ? PRODUCT_TYPE_COURSERUN : PRODUCT_TYPE_PROGRAM
+      )
+      const { inner } = await renderPage({
+        entities: {
+          basket
+        }
+      })
+      const runSelect = inner.find("select").at(0)
```

Comment:
Just to be a bit more specific, can you include the `run-selector` class name in this selector?